openff/bespokefit/executor/executor.py

```diff
@@ -349,114 +216,9 @@ def _stop(self):
         if self._remove_directory:
             shutil.rmtree(self._directory, ignore_errors=True)
 
-    @staticmethod
-    def submit(input_schema: BespokeOptimizationSchema) -> str:
```

Comment:
It'd be very bad to remove an API point that's used in the quickstart guide with no deprecation period. This would drop a fire in the lap of everyone who uses BespokeFit in a workflow and lose a lot of user trust. I'd recommend either:
* Adding a deprecation warning for ~3+ months 
* Not deprecating this at all, and just long-term redirecting to `client.submit_optimization`

Reply:
For now, I went with not deprecating it and just redirecting the methods to use the client.